src/main.rs

```diff
@@ -31,13 +33,13 @@ fn main() {
 fn run() -> Fallible {
     let app = App::new("Yarner")
         .version(crate_version!())
-        .about("Literate programming compiler\n  \
+        .about("Literate programming compiler\n\
                   https://github.com/mlange-42/yarner\n\
                 \n\
-                The normal workflow is:\n \
-                1) Create a project with\n    \
-                  > yarner create README.md\n \
-                2) Process the project by running\n    \
+                The normal workflow is:\n\
+                1) Create a project with\n\
+                  > yarner create README.md\n\
+                2) Process the project by running\n\
```

Comment:
Don't you think this was much better readable with indentations?

Reply:
Ah, I was assuming the indentation would come from the next line as with doc comments, but that is not actually the case. I'll go for raw string literals then as removing the indentation was certainly not my intent.

Reply:
Fixed using raw string literal.